### ISSUE TITLE
Small error in documentation for melspectrogram?

### ISSUE THREAD

**Issue by jaspervandorp** [librosa/librosa#1607]
Personally, I found the documentation for the "power" argument in melspectrogram a bit confusing, but this may also be due to the documentation:

```
"Exponent for the magnitude melspectrogram. e.g., 1 for energy, 2 for power, etc."
```

Shouldn't "energy" be "magnitude"?

Also, I would rephrase to:

```
"Exponent **applied to the spectrum before calculating the melspectrogram when the input is a time signal**, e.g. 1 for magnitude, 2 for power **(default)**, etc."
```

Finally, at the top, just for clarity I would rephrase:

```
"If a spectrogram input S is provided, then it is mapped directly onto the mel basis by mel_f.dot(S)."
```
to:

```
"If a spectrogram (power spectrum over time) input S is provided, then it is mapped directly onto the mel basis by mel_f.dot(S)."
```

**Comment by bmcfee** [comment]
Sure, I agree with the first couple of points.

For the last point, I don't think it's correct to refer to an arbitrary S input as a power spectrum, since it may have been computed with an exponent other than 2.

Would you like to make a PR for this?

**Comment by jaspervandorp** [comment]
> Sure, I agree with the first couple of points.
> 
> For the last point, I don't think it's correct to refer to an arbitrary S input as a power spectrum, since it may have been computed with an exponent other than 2.
> 
> Would you like to make a PR for this?

I agree with your last point. Maybe an alternative improvement would be to mention that the input should be a power spectrum if the user wants the same result compared to the default behavior if the input would have been a time signal (in that case, librosa transforms to a power spectrum first).

I'll make a PR as soon as I find the time.
